components/DatasetsList/index.tsx

```diff
@@ -179,20 +179,34 @@ export default function DatasetsList({
       }
 
       if (activeTopicFilter !== "All topics") {
-        filteredData = filteredData.filter(
-          (x) => x.topic === activeTopicFilter
+        filteredData = filteredData.filter((x) =>
+          x.topics.some((topic: string) =>
+            topic.includes(activeTopicFilter.toLowerCase().replaceAll(" ", "-"))
+          )
         );
       }
 
       if (activeSubtopicsFilter?.length > 0) {
         filteredData = filteredData.filter((x) =>
-          activeSubtopicsFilter.includes(x.subTopic)
+          x.topics.some((topic: string) => {
```

Comment:
if I'm understanding, this is recreating the "identifier" from the "title" of the topic?

not end of the world and we can manage (especially with data validation) but there's probably a world where they don't quite match so we'll need to look at this after the demo.